$NetBSD: patch-meson.build,v 1.3 2025/10/22 06:45:13 adam Exp $

Do not install file to set environment variables for systemd.
Reported upstream by email 20241227 (as a bug that systemd files are
installed on systems without systemd).

Make sure dependencies (opensc) do not get -pthread.
\todo Report upstream.

--- meson.build.orig	2025-10-19 13:06:54.000000000 +0000
+++ meson.build
@@ -275,10 +275,6 @@ install_data('etc/fr.apdu.pcsclite.metai
   install_dir : 'share/metainfo'
   )
 
-install_data('etc/pcscd',
-  install_dir : get_option('sysconfdir') / 'default'
-  )
-
 # generate config.h
 configure_file(output : 'config.h',
   configuration : conf_data)
@@ -323,7 +319,6 @@ pkg = import('pkgconfig')
 pkg.generate(
   extra_cflags : '-pthread',
   libraries : '-L${libdir} -lpcsclite',
-  libraries_private : '-pthread',
   subdirs : 'PCSC',
   version : meson.project_version(),
   name : 'PCSC Lite',
